plugins/vpp/abfplugin/vppcalls/vpp1908/abf_vppcalls_test.go

```diff
@@ -75,9 +75,9 @@ func TestAddABFPolicy(t *testing.T) {
 	Expect(req.Policy.ACLIndex).To(Equal(uint32(2)))
 	Expect(req.Policy.NPaths).To(Equal(uint8(2)))
 	Expect(req.Policy.Paths[0].SwIfIndex).To(Equal(uint32(5)))
-	Expect(req.Policy.Paths[0].NextHop[:4]).To(BeEquivalentTo(net.ParseIP("10.0.0.1").To4()))
+	Expect(req.Policy.Paths[0].Nh.Address.XXX_UnionData[:4]).To(BeEquivalentTo(net.ParseIP("10.0.0.1").To4()))
```

Comment:
Tests also should not use `XXX_` fields directly. Please use `GetIP4` and `GetIP6` fields.